hifi_vcpkg.py

```diff
@@ -283,9 +289,8 @@ def setupAndroidDependencies(self):
             dest = os.path.join(self.path, 'installed')
             url = self.readVar('EXTERNAL_VCPKG_ANDROID_URLS').split(';')
             # FIXME I don't know why the hash check frequently fails here.  If you examine the file later it has the right hash
```

Comment:
This seems to be happening everywhere now in github actions, except the self-hosted_debian-11_aarch64 build.

Reply:
There was a bug that causes all retries to fail once a single hash check failed (or in some cases even unconditionally it seems). After fixing that the builds are succeeding. Otherwise an occasional singular failure should be fine and should be taken care of by retries and fallbacks. Leaving this FIXME comment in since this particular case has not been tested.